doc/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -84,14 +99,15 @@ We will have to generate a YAML configuration file for configuring the 3D sensor
 
 
 Update the launch file
-----------------------
+++++++++++++++++++++++
 
 Add the YAML file to the launch script
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 You will now need to update the *sensor_manager.launch* file in the "launch" directory of your panda_moveit_config directory with this sensor information (this file is auto-generated by the Setup Assistant but is empty). You will need to add the following line into that file to configure the set of sensor sources for MoveIt! to use: ::
 
- <rosparam command="load" file="$(find panda_moveit_config)/config/sensors_kinect.yaml" />
+ <rosparam command="load" file="$(find panda_moveit_config)/config/sensors_kinect_pointcloud.yaml" />
 
+If you are using depthmap change the name of the yaml file to ``sensors_kinect_depthmap.yaml``.
```

Comment:
Depending if the [perception screen](https://github.com/ros-planning/moveit/issues/959) was to be merged first, adding this line would be automated.

Reply:
agreed. this tutorial can be simplified by relying on the perception screen